packages/ui/src/composites/wui-list-select/styles.ts

```diff
@@ -0,0 +1,37 @@
+import { css } from 'lit'
+
+export default css`
+  button {
+    display: flex;
+    column-gap: var(--wui-spacing-s);
+    align-items: center;
+    width: 336px;
```

Comment:
If I understand correctly idea for this component is to occupy full width of its parent in the modal right? So setting static width like this should be avoided.